api.swagger.yaml

```
@@ -1873,7 +1880,7 @@ paths:
         '403':
           description: The user does not have access to the requested node.
```

```suggestion
          description: The user does not have access to the requested content.
```
as long as you're doing this anyway